django_material_demo/Makefile

```diff
@@ -0,0 +1,41 @@
+# Put GENERATE_SECRET into files you want to generate secret
+# Usage: `make generate-secret FROM=/path/to/src TO=/path/to/dest`
+.PHONY: generate-secret
+generate-secret:
+	curl -s https://raw.githubusercontent.com/oursky/devsecops-secret/v1.0/generate-secret.sh \
+		| bash -s -- ${FROM} ${TO}
+
+.PHONY: secret
+secret:
+	make generate-secret FROM=.env.example TO=.env
+
+.PHONY: setup
+setup: secret
+	docker compose up --build --no-start
+	make migrations
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/django-admin/#makemigrations

> Creates new migrations based on the changes detected to your models.

The setup stage is meant to run right after a dev check out the repo. There should not be any model change, so it is not supposed to be run in setup stage.